packages/mantine/src/components/inline-confirm/InlineConfirm.tsx

```diff
@@ -21,12 +37,11 @@ export const InlineConfirm: InlineConfirmType = ({children}) => {
     const clearConfirm = () => setConfirmingId(null);
 
     return (
-        <InlineConfirmContext.Provider value={{confirmingId, setConfirmingId, clearConfirm}}>
+        <InlineConfirmProvider value={{confirmingId, setConfirmingId, clearConfirm}}>
             {prompt ?? children}
```

Comment:
The current behaviour make sense for a Menu or table actions. If the dev want to avoid this behaviour they could do it like this:
```tsx
<Group gap="xs">
  <SomeButtonThatWillAlwaysShow />
  <InlineConfirm> ... </InlineConfirm>
  <OtherButtonThatWillAlwaysShow />
</Group>
```